data/formatters/antivirus.yaml

```diff
@@ -236,3 +236,18 @@ message:
 - '{text}'
 short_message:
 - '{text}'
+---
+type: 'conditional'
+data_type: 'av:windows:defenderlog'
+message:
+- 'File Name: {filename}'
+- 'Hash: {sha256}'
+- 'User: {host_and_user}'
+- '{threatname}'
```

Comment:
maybe threat_name in case we want to expand the event data with other threat related attributes

Reply:
Done